### PR TITLE
Update efo-edit.owl

### PR DESCRIPTION
**Pull request by kallia-p**
Add new term Blood phopshate measurement for #942 and added CHEBI term 'phosphate ion'

### CODE REVIEW

File: src/ontology/efo-edit.owl [hunk 1]
```diff
@@ -202619,6 +202626,21 @@ AnnotationAssertion(rdfs:label efo:EFO_0010971 "response to metamizole"@en)
...
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "CMO:0000504") obo:IAO_0000115 efo:EFO_0010972 "The amount of phosphorus, measured as inorganic phosphate, in a specified sample of blood.")
+AnnotationAssertion(obo:IAO_0000117 efo:EFO_0010972 "Sam Lambert")
+AnnotationAssertion(efo:gwas_trait efo:EFO_0010972 "true"^^xsd:boolean)
+AnnotationAssertion(oboInOwl:hasDbXref efo:EFO_0010972 "ORCID:0000-0001-8222-008X")
+AnnotationAssertion(oboInOwl:hasDbXref efo:EFO_0010972 "PMID:33462484")
+AnnotationAssertion(Annotation(rdfs:comment "CMO:0000542") oboInOwl:hasExactSynonym efo:EFO_0010972 "Serum phosphate level")
+AnnotationAssertion(Annotation(rdfs:comment "CMO:0000504") oboInOwl:hasExactSynonym efo:EFO_0010972 "blood phosphate level")
+AnnotationAssertion(Annotation(rdfs:comment "https://www.ncbi.nlm.nih.gov/books/NBK310/") oboInOwl:hasExactSynonym efo:EFO_0010972 "serum inorganic phosphate concentration")
+AnnotationAssertion(Annotation(rdfs:comment "https://www.ncbi.nlm.nih.gov/books/NBK310/") oboInOwl:hasExactSynonym efo:EFO_0010972 "serum inorganic phosphorus concentration")
```
Comment (dosumis) [inline src/ontology/efo-edit.owl:202639]: All rdfs:comment here should be database_cross_reference

**Review by dosumis — CHANGES_REQUESTED** [review]
All looks good except need to switch comment to database_cross_reference for annotations on synonyms

File: src/ontology/efo-edit.owl (hunk 1 above)
Comment (kallia-p) [reply src/ontology/efo-edit.owl:202639]: Thanks all replaced by DbXref and commited by pushing this branch
AnnotationAssertion(Annotation(oboInOwl:hasDbXref "CMO:0000542") oboInOwl:hasExactSynonym efo:EFO_0010972 "Serum phosphate level")
AnnotationAssertion(Annotation(oboInOwl:hasDbXref "CMO:0000504") oboInOwl:hasExactSynonym efo:EFO_0010972 "blood phosphate level")
AnnotationAssertion(Annotation(oboInOwl:hasDbXref "https://www.ncbi.nlm.nih.gov/books/NBK310/") oboInOwl:hasExactSynonym efo:EFO_0010972 "serum inorganic phosphate concentration")
AnnotationAssertion(Annotation(oboInOwl:hasDbXref "https://www.ncbi.nlm.nih.gov/books/NBK310/") oboInOwl:hasExactSynonym efo:EFO_0010972 "serum inorganic phosphorus concentration")

**Review by dosumis — APPROVED** [review]

**Comment by dosumis** [comment]
@zoependlington  - want to take a quick look?  Looks fine to merge to me.

**Review by zoependlington — APPROVED** [review]
Looks good to me.

**Comment by dosumis** [comment]
@kallia-p  - please merge & delete branch.

**Pull request merged**